php/libraries/NDB_BVL_Instrument.class.inc

```diff
@@ -54,59 +38,59 @@ abstract class NDB_BVL_Instrument extends NDB_Page
     /**
      * Test name (short name), equivalent to test_names.Test_name
      *
-     * @access private
+     * @access public
      */
-    var $testName;
+    public $testName;
 
     /**
      * Instrument instance CommentID
      *
-     * @access private
+     * @access public
      */
-    var $commentID;
+    public $commentID;
 
     /**
      * Database table containing data referenced by $this->commentID
      *
-     * @access private
+     * @access protected
      */
-    var $table;
+    protected $table;
```

Comment:
This is changing the access level. `var` is equivalent to `public` in PHP, even if there's a comment saying otherwise. Changing this will break scripts that access the table (ie. lorisform_parser)